js/pie.js

```
@@ -46,9 +46,12 @@ function drawPie(pieData, color, cont) {
         .attr("class", "arc");
 
     var onclick = function (d, index) {
-        $('.scrollable-area').animate({
-            scrollTop: $("." + pieData[index].label).offset().top - tabsHeight
-        }, 200);
+        try {
+            $('.scrollable-area').animate({
+                scrollTop: $("." + pieData[index].label).offset().top - tabsHeight
+            }, 200);
+        } catch (e) {
+        }
     };
```

It is really bad practice to catch everything and ignore an exception. It is much better to understand what can cause an error. Take a look at the ticket and try to reproduce issue locally

---

@enigma91  I tried. it happens when we print n/a. When users click n/a it triggers scrollTop function, while we don't have objects, it prints exceptions. I can check if  `pieData[index].label class exists or not` but that if condition only checks for `n/a`. And when we check if classHas function in Jquery, it creates exception because `n/a` contains unallowed character `/` . try catch seemed more general.

---

Ok, sorry for the previous comment. I agree that it will be superfluous to add two additional checks in this case. But as for me I would still prefer superfluous solution with regexp + hasClass check, just to be sure, that my code will be predictable and easy to check

---

Now uses regex. There was special case for `/` character in regex. it allowed `n/a`, so i added indexof and existence of className conditions